src/core/plugins/react-17/root-injects.js

```diff
@@ -0,0 +1,5 @@
+export const render = (getSystem, getStore, getComponent, getComponents) => (domNode) => {
```

Comment:
Let's inject prettier comment so that the formatting of code is handled by prettier.

```suggestion
/**
 * @prettier
 */

export const render = (getSystem, getStore, getComponent, getComponents) => (domNode) => {
```